package exceptions.testthrows;

public class C {
	public static void f3(){
		throw new ArithmeticException();
	}
}
